### DISCUSSION TITLE
"There are outstanding issues with your GitHub account. Please contact GitHub Support for more information"

### DISCUSSION THREAD

**Discussion by ralphhack666** [community/community#70041] [GitHub Education]
### Hi everyone,

![IMG_20231013_012939](https://github.com/community/community/assets/145296973/113814a0-e312-407c-bec1-983f0f85530d)

I'm Rupam a college student pursuing B. Tech degree. I've been using GitHub for quite a while. I have tried to apply the Student Developer Pack to avail its facilities and resources. But my application has been rejected a few times and now when I'm trying to re-apply with no possible issues in my application it's showing "There are outstanding issues with your GitHub account. Please contact GitHub Support for more information" and won't let me proceed further with the application process.

Therefore I'm requesting GitHub Support to look into my application and resolve this issue at the earliest. Please inform me with my application status and path to solve this problem.

Thank you for your valuable time and consideration.

**Comment by JIASHIJIE530** [comment]
只需单击“GitHub 教育计划或权益”>“联系我们”，然后输入您的详细信息。工作人员会回复你。

**Reply by ralphhack666** [reply]
Yes, I went for it  in the first place but I couldn't open a ticket with my request. It said "Individual manual reviews and status updates are no longer provided" against "Application not approved (student)" request.

**Reply by JIASHIJIE530** [reply]
You can re-register a github account to apply, he will be fast, if you have no important things in this account, my friend is like this about 1 day to receive, click a adopt thank you,

**Reply by ralphhack666** [reply]
Yeah I can register a new account, only problem is the current account is connected to so many services especially Hacktoberfest. So It's better if I get the Student Developer pack in this account.
It's been a month and I'm still getting the same issue also can't reach out to GitHub Support. So what should I do now?

**Comment by BakulBd** [comment]
Did you get any solutions?

**Reply by ralphhack666** [reply]
NO

**Reply by BakulBd** [reply]
What are you doing then?
